packages/ember-data/lib/system/record_arrays/many_array.js

```diff
@@ -186,6 +187,34 @@ export default Ember.Object.extend(Ember.MutableArray, Ember.Evented, {
   },
 
   /**
+    Saves all of the records in the `ManyArray`.
+
+    Example
+
+    ```javascript
+    store.find('inbox', 1).then(function(inbox) {
+      inbox.get('messages').then(function(messages) {
+        messages.forEach(function(message) {
+          message.set('isRead', true);
+        });
+        messages.save()
+      });
+    });
+    ```
+
+    @method save
+    @return {DS.PromiseArray} promise
+  */
+  save: function() {
+    var promiseLabel = "DS: ManyArray#save " + get(this, 'type');
+    var promise = Ember.RSVP.all(this.invoke("save"), promiseLabel).then(function(array) {
+      return Ember.A(array);
```

Comment:
Updated `ManyArray.save()` and `RecordArray.save()` to return themselves in a separate commit.